common/men_at_arms_types/wc_regional_maa_types.txt

```diff
@@ -0,0 +1,189 @@
+﻿# standard costs
```

Comment:
Errors:
```
[13:17:23][pdx_persistent_reader.cpp:229]: Error: "Unexpected token: thoughness, near line: 1805
" in file: "common/men_at_arms_types/wc_cultural_maa_types.txt" near line: 1805
[13:17:23][pdx_persistent_reader.cpp:229]: Error: "Unexpected token: thoughness, near line: 1844
" in file: "common/men_at_arms_types/wc_cultural_maa_types.txt" near line: 1844
```
Errors:
```
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Left side and right side during comparison were of different types (left was 'character', right was 'culture')
  Script location: file:  file: common/script_values/00_men_at_arms_values.txt line: 154; like_arachnathid_trigger line: 2
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Left side and right side during comparison were of different types (left was 'character', right was 'culture')
  Script location: file:  file: common/script_values/wc_men_at_arms_values.txt line: 167; like_dragonhawk_trigger line: 3
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Left side and right side during comparison were of different types (left was 'character', right was 'culture')
  Script location: file:  file: common/script_values/wc_men_at_arms_values.txt line: 167; like_dragonhawk_trigger line: 4
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Left side and right side during comparison were of different types (left was 'character', right was 'culture')
  Script location: file:  file: common/script_values/wc_men_at_arms_values.txt line: 168; like_gryphon_trigger line: 6
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Left side and right side during comparison were of different types (left was 'character', right was 'culture')
  Script location: file:  file: common/script_values/wc_men_at_arms_values.txt line: 168; like_gryphon_trigger line: 7
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Failed to fetch a valid culture 'watewander'
  Script location: file:  file: common/script_values/wc_men_at_arms_values.txt line: 168; like_gryphon_trigger line: 8
[13:18:39][jomini_script_system.cpp:169]: Script system error!
  Error: Left side and right side during comparison were of different types (left was 'character', right was 'culture')
  Script location: file:  file: common/script_values/wc_men_at_arms_values.txt line: 168; like_gryphon_trigger line: 8
  ```

Reply:
I can't run the game, I think we already fixed this.